hadoop-ozone/interface-client/src/main/proto/OmClientProtocol.proto

```diff
@@ -1200,6 +1200,7 @@ message DeletedKeys {
 
 message PurgeKeysRequest {
     repeated DeletedKeys deletedKeys = 1;
+    optional string fromSnapshot = 2;
```

Comment:
`fromSnapshot` sounds too ambiguous to me. First I thought it is related to SnapDiff.

```suggestion
    // if not null, will purge keys in a snapshot DB instead of active
    optional string snapshotTableKey = 2;
```

And definitely need comments on the behavior where `snapshotTableKey` is null or not.